dashboards/management/commands/data_quality.py

```diff
@@ -0,0 +1,360 @@
+                elif voter.option == "other":
+                    voter_count_other += 1
+                else:
+                    print(voter.option)
+            # Checking to see if votes and vote counts match
+            if (voter_count_yes != 0 and voter_count_yes != total_yes) or (voter_count_no != 0 and voter_count_no != total_no):
+                total_votes_where_votes_dont_match_voters += 1
+    bill_vote_data = {
+        "total_votes_without_voters": total_votes_without_voters,
+        "total_votes_where_votes_dont_match_voters": total_votes_where_votes_dont_match_voters
+    }
+
+    return bill_vote_data
+
+
+def write_json_to_file(filename, data):
+    with open(filename, "w") as file:
+        file.write(data)
+
+# Example command
+# docker-compose run --rm django poetry run ./manage.py data_quality Virginia
+class Command(BaseCommand):
+    help = "export data quality as a json"
+
+    def add_arguments(self, parser):
+        parser.add_argument("state")
+        # parser.add_argument("sessions", nargs="*")
+        # parser.add_argument("--all-sessions", action="store_true")
+
+
+    def handle(self, *args, **options):
+        state = options["state"]
+        sessions = get_available_sessions(state)
+        chambers = get_chambers_from_abbr(state)
+        for session in sessions:
+            # Resets bills inbetween every session
+            bills = load_bills(state, session)
+            for chamber in chambers:
+                if bills.count() > 0:
+                    bills_per_session_data = total_bills_per_session(bills, chamber)
+                    average_num_data = average_number_data(bills, chamber)
+                    bill_version_data = bills_versions(bills, chamber)
+                    no_sources_data = no_sources(bills, chamber)
+                    bill_subjects_data = bill_subjects(bills, chamber)
+                    bill_vote_data = vote_data(bills, chamber)
+
+                    # overall_json_data = json.dumps({
+                    #     "chamber_name": chamber.classification,
+                    #     "bills_per_session_data": dict(bills_per_session_data),
+                    #     "average_num_data": dict(average_num_data),
+                    #     "bill_version_data": dict(bill_version_data),
+                    #     "no_sources_data": dict(no_sources_data),
+                    #     "bill_vote_data": dict(bill_vote_data),
+                    #     "bill_subjects_data": dict(bill_subjects_data)
+                    # })
+                    # filename = f"{state}_{session}_{chamber.classification}_data_quality.json"
+                    test = DataQualityDashboard.objects.update_or_create(
+                        state=state,
+                        session=session,
+                        chamber=chamber.classification,
+
+                        defaults={
+                            "total_bills":bills_per_session_data["total_bills"],
```

Comment:
a less repetitive way to write this would be to use unpacking sime the variable names are/can be the same

**bills_per_session_data,
**average_num_data,
**no_sources_data,

etc.

Reply:
Unpacked!